modules/machine-health-checks-resource.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * master/deploying-machine-health-checks.adoc
+
+[id='machine-health-checks-resource-{context}']
+= Sample `MachineHealthCheck` resource
+
+The `MachineHealthCheck` resource resembles the following YAML file:
+
+.MachineHealthCheck
+[source,yaml]
+----
+apiVersion: healthchecking.openshift.io/v1alpha1
+kind: MachineHealthCheck
+metadata:
+ name: example <1>
+ namespace: example <2>
+Spec:
+  Selector:
+    matchLabels:
+      sigs.k8s.io/cluster-api-cluster: <cluster_name> <3>
+      sigs.k8s.io/cluster-api-machine-role: <label> <4>
+      sigs.k8s.io/cluster-api-machine-type: <label> <4>
+      sigs.k8s.io/cluster-api-machineset: <cluster_name>-<label>-<AWS-zone> <5>
+----
+<1> IDK
+<2> IDK
+<3> Enter the name of your cluster.
+<4> Enter a label for the `MachinePool` that you want to check.
+<5> Enter the `MachineSet` to track in `<cluster_name>-<label>-<AWS-zone>`
+format. For example, `prod-node-us-east-1a`.
+
+
+
+////
```

Comment:
I saw both of these resources in a design doc. Are they still valid? If so, can we talk about how a user might employ them?

Reply:
There is no strict rule on the matching labels. The idea is to have one machine pool per one machineset. So in case a `MachineHealthCheck` resource is create to guard one machine pool, the matching labels should be exactly the same as machineset matching labels. Though, it's not forbidden to create one `MachineHealthCheck` resource per a set of machine pools. So the matching labels can be more exotic than.